### PR TITLE
Add option to configure DNS class to query

### PR DESCRIPTION
**Pull request by stklcode**
Addresses issue #102

### CODE REVIEW

File: zdns/main.go [hunk 1]
```diff
@@ -93,6 +95,24 @@ func main() {
 	// complete post facto global initialization based on command line arguments
 	gc.Timeout = time.Duration(time.Second * time.Duration(*timeout))
 	gc.IterationTimeout = time.Duration(time.Second * time.Duration(*iterationTimeout))
+	// class initialization
+	switch *class_string {
```
Comment (zakird) [inline zdns/main.go]: It would be nice to do a `strings.ToUpper()` here. Usability wise, it's a bit friendlier to also allow lowercase values.

**Review by zakird — APPROVED** [review]
Overall this looks great to me. I left one minor comment about accepting lowercase command line values.

**Comment by zakird** [comment]
@paul-pearce would you be willing to take a second glance at this? It looks good to me.

**Comment by stklcode** [comment]
Thanks. Just added the recommended case casting for the CLI parameter and also the common shotdhand values.

**Pull request merged**